<launch>
    <!-- Sim Time should be true for simulation -->
    <arg default="true" description="use_sim_time" name="use_sim_time"></arg>

    <!-- Declare parameter file path -->
    <arg name="param_file" default="$(find-pkg-share mono_depth)/config/mono_depth_cfg.yaml"/>

    <!-- Launch the GridVision Node with parameters -->
    <node pkg="mono_depth" exec="mono_depth" name="mono_depth_node" output="screen">
        <param name="use_sim_time" type="bool" value="$(var use_sim_time)"/>
        <param from="$(var param_file)" />
    </node>
</launch>
